dropwizard-parent-pom/pom.xml

```diff
@@ -252,11 +251,6 @@
       </dependency>
       <dependency>
         <groupId>io.dropwizard</groupId>
-        <artifactId>dropwizard-spdy</artifactId>
```

Comment:
Since Dropwizard now has a bom, in 0.8.5 i believe that wasn't yet the case, perhaps a parent pom isn't the best method to deal with this anymore? In most cases, teams can use the ecs-parent-pom and utilize the dropwizard-bom for dependencies. We may want our own bom to cover the additional dependencies we pull in everywhere...